docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '2.2'
+
+volumes:
+  boot:
+  state:
+  data:
+
+services:
+  provision:
+    image: alpine:3.13
+    command:
+      - sh
+      - -c
+      - |
+        if ! [ -f /mnt/boot/config.json ]; then
+          cp /config.json /mnt/boot/config.json
+        else
+          echo "INFO: Reusing already existing config.json in docker volume."
+        fi
+    volumes:
+      - boot:/mnt/boot
+      - ./config.json:/config.json
```

Comment:
is it possible to make this configurable?

I'm guessing that now we can only have one `config.json`. 

- What if someone wants to spin up several different containers belonging to separate apps?
- What if someone wants to emulate different devices belonging to same app?

Reply:
hmm right... we can make the local path configurable with an env var... `- ${CONFIG_JSON:-./config.json}:/config.json`

but if you want multiple containers you need to copy the entire service defintion over

Reply:
@rahul-thakoor I'm thinking if it's possible to make it work with the `scale: <replicas>` setting?

Reply:
never used that before @robertgzr

Reply:
probably not for this case... as it would just create `n` identical copies of the container

Reply:
which I think is fine... if you want to have devices in different apps you can either run `compose up` once per app or edit the composefile to include two `os` services

Reply:
see latest commit for the `multiple device, same app` option